README_GITFLEXIMOD.rst

```diff
@@ -18,33 +13,33 @@ To obtain the CTSM code you need to do the following:
 
    This will create a directory ``my_ctsm_sandbox/`` in your current working directory.
 
-#. Run the script **manage_externals/checkout_externals**. ::
+#. Run **./bin/git-fleximod update**. ::
 
-      ./manage_externals/checkout_externals
+      ./bin/git-fleximod update
 
-   The **checkout_externals** script is a package manager that will
+   **git-fleximod** is a package manager that will
    populate the ctsm directory with the relevant versions of each of the
    components along with the CIME infrastructure code.
 
 At this point you have a working version of CTSM.
 
 To see full details of how to set up a case, compile and run, see the CIME documentation at http://esmci.github.io/cime/ .
 
-More details on checkout_externals
-----------------------------------
+More details on git-fleximod
+----------------------------
 
 The file **Externals.cfg** in your top-level CTSM directory tells
-**checkout_externals** which tag/branch of each component should be
+**git-fleximod** which tag/branch of each component should be
 brought in to generate your sandbox. **Externals_CLM.cfg** is used similarly to point to the correct version of FATES (and possibly other CTSM-specific externals in the future); the below instructions referring to **Externals.cfg** also apply to modifying **Externals_CLM.cfg**.
 
-NOTE: checkout_externals will always attempt
+NOTE: git-fleximod will always attempt
 to make the working copy exactly match the externals description. If
 you manually modify an external without updating Externals.cfg, e.g. switch
```

Comment:
git-fleximod will not change a modified external without the --force option, instead it will warn you of local changes you need to resolve.